query_execution/tests/Foreman_unittest.cpp

```diff
@@ -252,15 +253,15 @@ class ForemanTest : public ::testing::Test {
   }
 
   inline const int getNumWorkOrdersInExecution(const QueryPlan::DAGNodeIndex index) const {
-    return foreman_->queued_workorders_per_op_[index];
+    return foreman_->query_exec_state_->getNumQueuedWorkOrders(index);
```

Comment:
I would rather have ``foreman_`` be there because it clearly tells the reader where ``query_exec_state_`` is coming from.